content/master/concepts/pods.md

```diff
@@ -0,0 +1,321 @@
+
+{{< hint "note" >}}
+Kubernetes defines the initial rate of 100 times per second. This isn't
+configurable.
+{{< /hint >}}
+
+All core Crossplane components share the retry rate. Each Provider
+implements their own retry rate. 
+
+##### Number of reconcilers
+
+The second value `--max-reconcile-rate` defines is the number of threads
+created to reconcile resources. Each thread manages the reconciliation of a
+specific resource. If there are more resources than configured threads,
+remaining resources must wait until a new thread is available. 
+
+Changing `--max-reconcile-rate` changes the number of threads Crossplane creates
+when more than one resource need reconciliation.
+
+Read the [Change Pod Settings]({{<ref "#change-pod-settings">}}) section for
+instructions on applying these settings. 
+
+<!-- vale Microsoft.HeadingAcronyms = NO -->
+<!-- allow 'RBAC' since that's the name -->
+## RBAC manager pod
+<!-- vale Microsoft.HeadingAcronyms = YES -->
+The Crossplane RBAC manager pod automates required Kubernetes RBAC permissions
+for Crossplane and Crossplane Providers. 
+
+{{<hint "note" >}}
+Crossplane installs and enables the RBAC manager by default.
+Disabling the RBAC manager requires manual Kubernetes permissions definitions
+for proper Crossplane operations. 
+
+The 
+[RBAC manager design document](https://github.com/crossplane/crossplane/blob/master/design/design-doc-rbac-manager.md) 
+provides more comprehensive details on the Crossplane RBAC requirements.
+{{< /hint >}}
+
+<!-- vale Microsoft.HeadingAcronyms = NO -->
+<!-- allow 'RBAC' since that's the name -->
+### RBAC init container
+<!-- vale Microsoft.HeadingAcronyms = YES -->
+
+The RBAC manager requires the `CompositeResourceDefinition` and
+`ProviderRevision` resources to be available before starting.
+
+The RBAC manager init container waits for these resources before starting the 
+main RBAC manager container. 
+
+
+<!-- vale Microsoft.HeadingAcronyms = NO -->
+<!-- allow 'RBAC' since that's the name -->
+### RBAC manager container
+<!-- vale Microsoft.HeadingAcronyms = YES -->
+
+The RBAC manager container preforms the following tasks:
+* creating and binding RBAC roles to Provider Service Accounts, allowing 
+  them to control their managed resources
+* allowing the `crossplane` Service Account to create managed resources
+* creating and binding RBAC roles for Crossplane composite resources
+
+When installing a Provider their service account needs permissions to create and 
+manage managed resources. 
+
+When creating composite resources the Crossplane service account needs
+permissions for the newly created API endpoints. 
+
+The RBAC manager container manages these functions. 
+
+The RBAC manager pod supports two policy options:
+* `All` - The default policy, configures and manages all RBAC capabilities.
+* `Basic` - Doesn't enable namespaced RBAC roles.
+
+Using the `Basic` policy option prevents creating namespaced Crossplane RBAC
```

Comment:
I know you're attempting to be succinct here, but if you could briefly mention what the namespaced Crossplane RBAC roles are for I think it would help contextualize why folks would potentially use `Basic`.

Something like:

> When the RBAC manager uses the `All` policy it will create a series of opinionated RBAC roles in each namespace. You can bind these RBAC roles to users to grant them access to Crossplane types within a namespace.